_bookdown.yml

```diff
@@ -2,6 +2,6 @@ book_filename: "Peace2022"
 language:
   ui:
     chapter_name: ""
-delete_merged_file: false
-output_dir: "docs/cfe4dfac2f8c88e24c52504a2eed0461ba3441ea"
```

Comment:
again - this is a password protected website so this facilitate the requirement for a password. This changed caused a merge conflict when it shouldn't of...

 I documented elsewhere how it all works the other day.  see if you can find it. I don't need to hear about it if you can find it. Let me know if you can't and I will point you in the right direction

Reply:
I'm not sure what you're referring to. Should I revert this commit?

Reply:
i may have wrote it all down and then closed a million google windows and lost it. that happenned at some point.  I will put it in the `mybookdown-template` repo as an issue and  in the `readme` as that is a good place for it. Its pretty sick how the password protect sites work and it should be easy for us to find how it functions under the hood

Reply:
Actually - I documented it here.  I'll just link to that in the mybookdown-template repo